include/flang/Lower/PFTBuilder.h

```diff
@@ -26,44 +39,18 @@ struct Program;
 struct ModuleLikeUnit;
 struct FunctionLikeUnit;
 
-// TODO: A collection of Evaluations can obviously be any of the container
-// types; leaving this as a std::list _for now_ because we reserve the right to
-// insert PFT nodes in any order in O(1) time.
-using EvaluationCollection = std::list<Evaluation>;
+// Using std::list for a list of Evaluations preserves the option to do
+// O(1) time insertions anywhere.
+using EvaluationList = std::list<Evaluation>;
```

Comment:
This is game playing with censors, not to necessarily disparage censorship, which can have value on occasion.  It was a good idea to make this a using declaration, since that reduces the downstream impact of choosing something else.  But this really is a list, not an unordered collection.  I would consider changing it to a vector (still a list in the abstract), but you and/or Jean pointed out that that can't be done due to pointer invalidation considerations.  If someone suggests an llvm list replacement, that should be an acceptable one line change.  Someone said dequeue is valid, but that just seems goofy to me.  So this seems to me to be a reasonable choice for the moment.

Reply:
Hmm. I seems like it was `EvalList` then it was changes to `EvalCollection` and now it's back to `EvalList`. I may be mistaken though.

Reply:
Almost - the net change is EvalList --> EvaluationList.  The children of a tree node form a list, so I think that's ok.  (games)